tests/mixed_int8/test_mixed_int8.py

```diff
@@ -40,6 +41,13 @@
 from transformers.utils.versions import importlib_metadata
 
 
+if is_accelerate_available():
+    from accelerate import Accelerator
+    from accelerate.logging import get_logger
+
+    logger = get_logger(__name__)
+    _ = Accelerator()
```

Comment:
Let's use the `PartialState` here instead as that's what it's designed for
```suggestion
    _ = PartialState()
```

(And add `from accelerate import PartialState`)